src/actions/targetActions.js

```diff
@@ -50,3 +51,45 @@ export const loadTargets = () =>
         throw err;
       });
   };
+
+export const selectedTarget = target => ({
+  type: types.SELECTED_TARGET,
+  target
+});
+
+export const selectTarget = ({ target }) =>
+  dispatch =>
+    dispatch(selectedTarget(target));
+
+export const deselectedTarget = () => ({
+  type: types.DESELECT_TARGET
+});
+
+export const deselectTarget = () =>
+  dispatch =>
+    dispatch(deselectedTarget());
+
+export const deleteTarget = () => ({
+  type: types.DELETE_TARGET
+});
+
+export const deleteTargetSuccess = () => ({
+  type: types.DELETE_TARGET_SUCCESS
+});
+
+export const deleteTargetFail = () => ({
```

Comment:
I would name it failure
```suggestion
export const deleteTargetFailure = () => ({
```
Same for the action type